ISISComputingGroup/ibex_gui
# Ticket 2607: Galil specific opi

### Description of work

This is an OPI that contains all the parameters that are specific to a galil over other motors.

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/2607

### Acceptance criteria

Will also need:
* https://github.com/ISISComputingGroup/EPICS-ioc/pull/203
* https://github.com/ISISComputingGroup/EPICS-galil/pull/16

To test:
* Point to a real galil 
* Confirm that all fields are connected and contain sensible values.
* Confirm the steps per unit labels change with updating motor units
* Confirm the inverse motor/encoder steps can be set

---

#### Code Review


### Final Steps
